### PR TITLE
Normalize and pil link fix

### PR DESCRIPTION
**Pull request by kohr-h**
Update for a broken link to the PIL docs, and a more efficient version of the `normalize` transform.

### CODE REVIEW

**Comment by soumith** [comment]
thank you!

**Comment by vfdev-5** [comment]
@kohr-h looks good, as you are on `F.normalize`, maybe it would be nice to add a notice in the documentation to cover #517 ? 
cc @soumith

**Comment by kohr-h** [comment]
@vfdev-5 What would be the intention here, efficiency or keeping input intact? Looking at other transforms, they never explicitly copy, but probably most of them can't work in-place anyway.

I can add a notice in the docs, sure.

**Comment by vfdev-5** [comment]
@kohr-h I would say that efficiency is better and among all other transforms `F.normalize` is the only transform working on `torch.Tensor` and returning `torch.Tensor`, so just add a notice that it works in-place can be enough.
